python4j/python4j-core/src/test/java/PythonJobTest.java

```diff
@@ -0,0 +1,286 @@
+/*******************************************************************************
+ * Copyright (c) 2015-2018 Skymind, Inc.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+
+@javax.annotation.concurrent.NotThreadSafe
+public class PythonJobTest {
+
+    @Test
+    public void testPythonJobBasic() throws Exception{
+        PythonContextManager.deleteNonMainContexts();
+
+        String code = "c = a + b";
+        PythonJob job = new PythonJob("job1", code, false);
+
+        List<PythonVariable> inputs = new ArrayList<>();
+        inputs.add(new PythonVariable<>("a", PythonTypes.INT, 2));
+        inputs.add(new PythonVariable<>("b", PythonTypes.INT, 3));
```

Comment:
Usability thought unrelated to this test - what about calling these `PyVariable` and `PyTypes` instead? Less for users to type but still reasonably clear.

Reply:
Sure about this? Currently `Python*` classes belong to Python4j and `Py*` classes/methods belong to CPython.